### PR TITLE
Fixup issue #295

### PR DESCRIPTION
**Pull request by JellyBellyDev**
[fixup/issues-295 08555d1] Fixup issue #295

### CODE REVIEW

File: src/Badge/Model/UseCase/CreateComposerLockBadge.php [hunk 1]
```diff
@@ -64,8 +64,8 @@ public function createComposerLockBadge(string $repository, string $format = 'sv
             'HEAD',
             $repo.'/blob/'.$package->getDefaultBranch().'/composer.lock',
             [
-                RequestOptions::TIMEOUT => 2,
-                RequestOptions::CONNECT_TIMEOUT => 1,
+                RequestOptions::TIMEOUT => 8,
```
Comment (AlessandroMinoccheri) [inline src/Badge/Model/UseCase/CreateComposerLockBadge.php:67]: Could it be better to put these values into constants?
So we could also specify if they are seconds or other to improve readability, what do you think?

Comment (JellyBellyDev) [reply src/Badge/Model/UseCase/CreateComposerLockBadge.php:67]: make sense...
What name would you give to constants?

Comment (AlessandroMinoccheri) [reply src/Badge/Model/UseCase/CreateComposerLockBadge.php:67]: For example something like 
```
TIMEOUT_SECONDS
CONNECT_TIMEOUT_SECONDS
```

make sense?

Comment (JellyBellyDev) [reply src/Badge/Model/UseCase/CreateComposerLockBadge.php]: yep!

**Pull request merged**